docs/src/introduction/getting-started.md

```diff
@@ -1 +1,131 @@
+    database_filepath = "store.sqlite3"
+   ```
+
+3. **Build & install the Client:** install the client using cargo:
+
+   ```shell
+   cargo install --features testing --path .
+   ```
+
+   you should now be able to use the following command:
+
+   ```shell
+   miden-client --help
+   ```
+
+## Step 2: Setting-up the Miden-Client
+
+1. **Creating new accounts:** To be able to interact with the Miden node we will need to generate accounts. For this example we will be generating 2 accounts: A `basic-immutable` account and a `fungible-faucet`. You can generate new accounts using the following command:
+
+   ```shell
+   miden-client account new basic-immutable
+   miden-client account new fungible-faucet [...]
+   ```
+
+   Please refer to the documentation of the CLI
+
+2. **Listing accounts:** To view the newly created accounts we can run the following command:
+
+   ```shell
+   miden-client account -l
+   ```
+
+3. **Syncing node state:** The client needs to periodically query the node to receive updates about entities that might be important in order to run transactions. The way to do so is by running the `sync` command:
+
+   ```shell
+   miden-client sync-state -s
+   ```
+
+## Step 3: Minting an asset
+
+Since we have now synced our local view of the blockchain and have account information, we are ready to execute and submit tranasctions. For a first test, we are going to mint a fungible asset for a regular account.
+
+```shell
+miden-client tx new mint <regular-account-ID> <faucet-account-id> 1000
+```
+
+This will execute, prove and submit a transaction that mints assets to the node. The account that executes this transaction will be the faucet as was defined in the node's configuration file. In this case, it is minting `1000` fungible tokens to `<regular-account-ID-A>`.
+
+This will add a transaction and an output note (containing the minted asset) to the local store in order to track their lifecycles. You can display them by running `miden-client tx list` and `miden-client input-notes list` respectively. If you do so, you will notice that they do not show a `commit height` even though they were submitted to the operator. This is because our local view of the network has not yet been updated. After updating it with a `sync`, you should see the height at which the transaction and the note containing the asset were committed. This will allow us to prove transactions that make use of this note, as we can compute valid proofs that state that the note exists in the blockchain.
+
+## Step 4: Consuming the note
+
+After creating the note with the minted asset, the regular account can now consume it and add the tokens to its vault. You can do this the following way:
+
+```bash
+miden-client tx new consume-note <regular-account-ID-A> <input-note-ID>
+```
+
+This will consume the input note ID, which you can get by listing them as explained in the previous step. You will now be able to see the asset in the account's vault by running:
```

Comment:
`This will consume the input note ID` -> `This will consume the input note`?

Reply:
@igamigo this would also apply to the miden-client readme.